srcpkgs/frescobaldi/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'frescobaldi'
+pkgname=frescobaldi
+version=3.1.3
```

Comment:
Is a release with python 3.10 fixes expected? It'd be nice to see one from upstream.

Reply:
Yes, the patches included here are taken from commits to the upstream repository that will eventually end up in a release.